pkg/controller/controller_test.go

```diff
@@ -7,22 +7,53 @@ import (
 	"github.com/stretchr/testify/require"
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 
 	"github.com/infracloudio/botkube/pkg/config"
 	"github.com/infracloudio/botkube/pkg/utils"
-	testutils "github.com/infracloudio/botkube/test/e2e/utils"
 )
 
 // TODO: Tests moved out straight from E2E test package with minimal changes.
```

Comment:
is it still valid?

Reply:
This is still valid, as I didn't write additional test cases for the unit tests. This comment links to the test epic, co I think it is fine to keep it as it is. I believe will gradually rework unit tests when we will touch these packages, so the epic would be kept open.